### PR TITLE
VALIDATE SSHIB

### PR DESCRIPTION
**Pull request by kingsaiyanSSHIB**
# Validate [SSHIB](https://solscan.io/token/6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/shibonsolana/status/1741126315565261245?s=46
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/solana-shib / https://coinmarketcap.com/currencies/solana-shib/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by Renonoy25** [comment]
Mint address: 6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA
Symbol: SSHIB
Token Decimals: 5
Token icon: https://gateway.irys.xyz/3pwcTcBUEAE9nHRmiN1gyHvnSAaiqEP4PmIs9tZu1yQ
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by SSHIBCAP** [comment]
$SSHIB CAPTAIN HERE 🔥🚀🐋 LETSS GOO!!

**Comment by siddhere03** [comment]
Mint address: 6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA
Symbol: SSHIB
Token Decimals: 5
Token icon: https://gateway.irys.xyz/3pwcTcBUEAE9nHRmiN1gyHvnSAaiqEP4PmIs9tZu1yQ
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Sshibmaster** [comment]
Sshib we will Prefail❤️

**Comment by Ndiouga93** [comment]
Mint address: 6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA
 Symbol: SSHIB
 Token Decimals: 5
 Token icon: https://gateway.irys.xyz/3pwcTcBUEAE9nHRmiN1gyHvnSAaiqEP4PmIs9tZu1yQ
 On-chain metadata matches
 No duplication on Jupiter's strict list

**Comment by 0xSoju2** [comment]
This is wrong, you guys need to add to the token-list json

**Comment by siddhere03** [comment]
> This is wrong, you guys need to add to the token-list json

@0xSoju2 can you point me to the token-list json you are mentioning so I can add it there" in this PR thread you guys made https://github.com/jup-ag/token-list/pull/878

**Comment by kingsaiyanSSHIB** [comment]
> can you point me to the token-list json you are mentioning so I can add it there" in this PR thread you guys made

File: validated-tokens.csv [hunk 1]
```diff
@@ -1057,5 +1057,6 @@ PONKE,PONKE,5z3EqYQo9HiCEs3R84RCDMu2n7anpDMxRhdK8PSWmrRC,9,https://i.imgur.com/q
 CATWIFHAT,CIF,G3vWvAaXPHCnncnyAbq6yBRXqfRtEV3h7vExzasZeT6g,6,https://i.postimg.cc/hGBgdxxj/Cif200x200.png,true
 AI,AI,99ouK5YUK3JPGCPX9joNtHsMU7NPpU7w91JN4kdQ97po,9,https://gateway.irys.xyz/HnDwG54RZvJhTD2PX-mmrD2jwC9MIdxKjW2hL3M8jQc,true
 catwifhat,$CWIF,7atgF8KQo4wJrD5ATGX7t1V2zVvykPJbFfNeVf1icFv1,2,https://i.postimg.cc/d1QD417z/200x200logo-copy.jpg,true
+SSHIB,$SSHIB,6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA,5,https://solshib.space/resources/images/icons/logo.png
```
Comment (hankc97) [inline validated-tokens.csv:1060]: SSHIB,SSHIB,6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA,5,https://node1.irys.xyz/3pwcTcBUEAE9nHRmiN1gyHvnSAaiqEP4PmIs9tZu1yQ,true

**Comment by hankc97** [comment]
@0xSoju2 should be fixed now

**Comment by DanielCherman** [comment]
Mint address: 6VHL2vMKgrF1YQFSv29Rs1pj9VCRK29bD11NtDqerqHA
Symbol: SSHIB
Token Decimals: 5
Token icon: https://gateway.irys.xyz/3pwcTcBUEAE9nHRmiN1gyHvnSAaiqEP4PmIs9tZu1yQ
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by 0xSoju2 — APPROVED** [review]

**Pull request merged**